### PR TITLE
Use bqetl CLI for running baseline queries

### PR DESCRIPTION
**Pull request by scholtzan**
Depends on https://github.com/mozilla/bigquery-etl/pull/2027

I think this and bigquery-etl are the only places the scripts are used?

### CODE REVIEW

File: dags/copy_deduplicate.py
Comment (scholtzan) [inline dags/copy_deduplicate.py:183]: This seems to work for https://github.com/mozilla/telemetry-airflow/blob/2407ba20bf5e6fc35fd5ab3d3194d6d7302bc5a1/dags/stripe.py#L20

**Comment by jklukas** [comment]
This looks good. Will hold off on approving until https://github.com/mozilla/bigquery-etl/pull/2027 is settled, in case we need a change to the CLI invocation here.

**Review by jklukas — APPROVED** [review]
r+wc

We should be careful in ordering of events when we merge this. I believe we should wait for https://github.com/mozilla/bigquery-etl/pull/2027 to be merged and verify that the `generated-sql` content matches our expectations before merging this one.

**Pull request merged**
